### PR TITLE
Code challenge

### PR DESCRIPTION
**Pull request by sgwesley**

### CODE REVIEW

File: step2/CodeReview.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ struct ThirdPartyAVSoftware
     std::wstring Name;
     std::wstring Description;
     std::wstring DefinitionUpdateTime;
-    std::string DefinitionStatus;
+    std::string DefinitionStatus; // wouldnt be wstring?
```
Comment (sgwesley) [inline step2/CodeReview.cpp:19]: Suggestion: I would change it to `wstring` in order to follow the pattern.

File: step2/CodeReview.cpp [hunk 2]
```diff
@@ -97,6 +98,7 @@ bool queryWindowsForAVSoftwareDataWSC(std::map<std::wstring, ThirdPartyAVSoftwar
         hr = PtrProduct->get_SignatureStatus(&ProductStatus);
         if (FAILED(hr))
         {
+			// release PtrProduct
```
Comment (sgwesley) [inline step2/CodeReview.cpp:101]: Same situation mentioned above.

File: step2/CodeReview.cpp [hunk 3]
```diff
@@ -106,6 +108,7 @@ bool queryWindowsForAVSoftwareDataWSC(std::map<std::wstring, ThirdPartyAVSoftwar
         hr = PtrProduct->get_ProductStateTimestamp(&PtrVal);
         if (FAILED(hr))
         {
+			// release PtrProduct
```
Comment (sgwesley) [inline step2/CodeReview.cpp:111]: Same situation mentioned above.

**Comment by TarabullNinja** [comment]
@sgwesley Please put all your work inside the screening exam, notes on the pull request like such will be difficult the translate to the team. Let me know once you are ready and I will close out this PR and move your exam along.

**Comment by sgwesley** [comment]
> @sgwesley Please put all your work inside the screening exam, notes on the pull request like such will be difficult the translate to the team. Let me know once you are ready and I will close out this PR and move your exam along.

@TarabullNinja Sorry my misunderstanding. I have deleted my comments and put it into the code.

**Comment by TarabullNinja** [comment]
@sgwesley Perfect, will move your submission along now, thank you!

**Pull request closed without merging**